manager-bundle/composer.json

```diff
@@ -59,6 +59,10 @@
         "phpunit/phpunit": "^7.5",
         "symfony/phpunit-bridge": "^4.3"
     },
+    "suggest": {
+        "contao/tcpdf-bundle": "To export articles as PDF files.",
+        "contao/mobile-page-layout-bundle": "To re-gain the functionality of mobile page layouts (not recommended)."
```

Comment:
What is the point of suggesting packages that we do not recommend to add?

Reply:
well they come from Contao and might complete a setup 🤷‍♂. I just found tcpdf-bundle as a suggestions of core-bundle, but most people will only have/see manager-bundle.

Reply:
That does not answer my question, does it? 

There is no point in suggesting contao/mobile-page-layout-bundle if we simultaneously say "not recommended". Then we should not suggest it at all.

>  just found tcpdf-bundle as a suggestions of core-bundle, but most people will only have/see manager-bundle.

I am fine with suggesting contao/tcpdf-bundle in the manager bundle as well.